api/schemas/input/file.json

```diff
@@ -3,7 +3,7 @@
   "type": "object",
   "properties": {
     "name":           { "type": "string" },
-    "type":           { "type": "string" },
+    "type":           { "type": ["string", "null"] },
```

Comment:
I am afraid, I didn't have the time to partecipate in that discussion. Will remove though.